docs/END-TO-END-TESTING.md

```diff
@@ -0,0 +1,103 @@
+    export BROWSERSTACK_ACCESS_KEY="" # Your BrowserStack access key
+    export BROWSERSTACK_PROJECT_NAME="plio"
+    ```
+
+3. Once the env variables are added, restart your terminal or run one of the following command:
+    ```sh
+    source ~/.bash_profile
+    # or
+    source ~/.zshrc
+    ```
+
+### Getting started
+1. To run a test case locally, execute the following command from the project root folder:
+    ```sh
+    # syntax
+    testcafe browser ./path/to/test.spec.js
+
+    # example
+    testcafe chrome ./tests/integration/pages/login.spec.js
+    ```
+2. To list all the browsers available, run the following command:
+    ```sh
+    # command to see local browsers
+    testcafe --list-browsers
+
+    # command to see BrowserStack browsers
+    testcafe -b browserstack
+    ```
+
+
+### Environment variables
+Setup the following environment variables in your `~/.bash_profile` or `~/.zshrc` file if you're trying to run testcafe locally with BrowserStack. Also, set them up on the Continuous Integration environment using GitHub secrets.
+
+#### `BROWSERSTACK_BASE_URL`
+The base url of the application registered with BrowserStack. Should be the frontend application URL.
+
+#### `BROWSERSTACK_USERNAME`
+Required to connect to your BrowserStack account. Get this information from your BrowserStack account settings.
+
+#### `BROWSERSTACK_ACCESS_KEY`
+Required to connect to your BrowserStack account. Get this information from your BrowserStack account settings.
+
+#### `BROWSERSTACK_PROJECT_NAME`
+Name of the BrowserStack project. Set to `plio`. It allows to name the build, which is useful for keeping track of test runs on the BrowserStack dashboard.
+
+### Environment variables (only for Continuous Integration)
+The following environment variables are only required for the Continuous Integration environment. These are used to install and build the frontend and backend application. Set these up using GitHub secrets only and not in the `.env` files.
+
+#### `GOOGLE_OAUTH2_CLIENT_ID`
+Google refresh token retrieved in the pre-requisites.
+
+#### `GOOGLE_OAUTH2_CLIENT_SECRET`
+Google client ID retrieved in the pre-requisites.
+
+#### `GOOGLE_OAUTH2_REFRESH_TOKEN`
+Google client secret retrieved in the pre-requisites.
+
+#### `BACKEND_CONVERT_SOCIAL_AUTH_TOKEN_URL`
+The convert token URL in the Plio backend repo that converts the Social Auth Token and returns a valid Auth Token. By default, the value is http://0.0.0.0:8001/auth/convert-token.
+
+#### `BACKEND_DEFAULT_OAUTH2_CLIENT_ID`
+The client id for Plio backend application. More details in [Plio Backend ENV.md](https://github.com/avantifellows/plio-backend/blob/master/docs/ENV.md#default_oauth2_client_id).
+
+#### `BACKEND_DEFAULT_OAUTH2_CLIENT_SECRET`
+The client secret for Plio backend application. More details in [Plio Backend ENV.md](https://github.com/avantifellows/plio-backend/blob/master/docs/ENV.md#default_oauth2_client_secret).
```

Comment:
why not use `VUE_APP_BACKEND_API_CLIENT_ID` and `VUE_APP_BACKEND_API_CLIENT_ID` which are already being set instead of creating two new env variables doing the exact same thing?